### PR TITLE
Yel 2222

### PR DESCRIPTION
**Pull request by duncan9562**

### CODE REVIEW

**Review by JWanderema — APPROVED** [review]
LGTM

File: README.md [hunk 1]
```diff
@@ -1 +1,63 @@
...
+## Introduction
+This app archive's tables in Athena and is designed to run in Kubernetes. It utilises Kubernetes `cronjob` that triggers based on the value of `KUBE_SCHEDULE`.
+
+When triggered a job is created which starts a POD, and a container with the image from Quay:-
+
+It currently only supports a rule to archive data that is older than 2 months + current.
+
+A CSV file must exist in S3 which contains the database name, table name and S3 location of the data.
+
+**Note:** The `_arhicve` table must exist prior to running.
```
Comment (ianrelf) [inline README.md]: Typo: `_arhicve`

File: README.md [hunk 2]
```diff
@@ -1 +1,63 @@
...
+
+The layout of the CSV file is as follows:-
+
+|  database_name  |  table_name  |      s3_location      |
+| --------------- | ------------ | --------------------- |
+|    database1    |    table1    | s3-a-location/in/s3   |
+|    database1    |    table2    | s3-a-location/in/s3   |
+|    database1    |    table3    | s3-a-location/in/s3   |
+|    database2    |    table1    | s3-a-location/in/s3   |
+|    database3    |    table1    | s3-a-location/in/s3   |
```
Comment (ianrelf) [inline README.md]: I think you should provide a concrete example (this will help with whether or not to include "S3://")

File: README.md (hunk 1 above)
Comment (duncan9562) [reply README.md]: Updated in the latest commit.

File: README.md (hunk 2 above)
Comment (duncan9562) [reply README.md]: Updated in the latest commit.

**Review by ianrelf — APPROVED** [review]

**Pull request merged**